src/main/webapp/sass/_core.scss

```diff
@@ -3076,10 +3076,102 @@ div.border {
 	font-style:italic;
 }
 
-#body .fullWidthAlert {
-	@include border-radius(0px);
-	box-shadow: none;
-	padding: 7px 15px;
+#body .synapseAlert {
```

Comment:
See https://sagebionetworks.jira.com/browse/SWC-5354 for screenshots of what this looks like in practice.